### ISSUE TITLE
how to save a model with weights and paramters 

### ISSUE THREAD

**Issue by aLLLiyyy** [tensorflow/tensorflow#43643]
when i create a model and compiled it and call the model.fit() and then i saved the model called model.save(path)
then i tried to load my saved module and i get the model correctly but  it cannot load the ckpt files so the evaluate result showed accuracy is around 10%,  i mean, how can i load module without training it again?

**Comment by ravikyram** [comment]
@aLLLiyyy 

Please, go through this [tutorial ](https://www.tensorflow.org/tutorials/keras/save_and_load?hl=en)and see if it helps you.Thanks!

**Comment by ymodak** [comment]
You need to load the weights from the checkpoint and re-evaluate.
How are you saving the checkpoints?
If using [`tf.keras.callbacks.ModelCheckpoint`](https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/ModelCheckpoint) api you can use `model.load_weights` method

```python
# Loads the weights
model.load_weights(checkpoint_path)

# Re-evaluate the model
loss,acc = model.evaluate(test_images,  test_labels, verbose=2)
print("Restored model, accuracy: {:5.2f}%".format(100*acc))
```

**Comment by aLLLiyyy** [comment]
@ymodak thanks for your reply,but i think you missread what i mean，i already saved the checkpoint as you said using the ModelCheckpoint api ,but i still need to create that model explicitly,do you know a method that i can save the full model and weights directly as a frozen pb file, then what should i do?thank you!

**Comment by ymodak** [comment]
This [article](https://leimao.github.io/blog/Save-Load-Inference-From-TF2-Frozen-Graph/) can be a good starting point to convert your model into `.pb` format.

**Comment by aLLLiyyy** [comment]
i d like to read it soon,
